motionplan/tpSpaceRRT_test.go

```diff
@@ -0,0 +1,215 @@
+//go:build !windows
+
+package motionplan
+
+import (
+	"context"
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	pb "go.viam.com/api/component/arm/v1"
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/motionplan/tpspace"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+)
+
+func TestPtgRrt(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	ackermanFrame, err := NewptgFrame("test")
+	test.That(t, err, test.ShouldBeNil)
+
+	goalPos := spatialmath.NewPose(r3.Vector{X: 50, Y: 10, Z: 0}, &spatialmath.OrientationVectorDegrees{OZ: 1, Theta: 180})
+
+	opt := newBasicPlannerOptions()
+	opt.SetGoalMetric(NewPositionOnlyMetric(goalPos))
+	opt.DistanceFunc = SquaredNormNoOrientSegmentMetric
+	opt.GoalThreshold = 10.
+	mp, err := newTPSpaceMotionPlanner(ackermanFrame, rand.New(rand.NewSource(42)), logger, opt)
+	test.That(t, err, test.ShouldBeNil)
+	tp, ok := mp.(*tpspaceRRTMotionPlanner)
+	test.That(t, ok, test.ShouldBeTrue)
+
+	_, err = tp.plan(context.Background(), goalPos, nil)
+	test.That(t, err, test.ShouldBeNil)
```

Comment:
I think testing the output of the plan is important here.  Even if its just that the length of the plan is at least 2